provider/keycloak_ldap_user_federation.go

```diff
@@ -55,7 +54,7 @@ func resourceKeycloakLdapUserFederation() *schema.Resource {
 			"import_enabled": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     true,
```

Comment:
I think there might be an issue here with fields that default to true (like `import_enabled)`. When the provider does the mapping from resource data to api model, the initial `data.Get("import_enabled")` [will return false (the empty value for booleans)](https://github.com/hashicorp/terraform/blob/15dda6335876145e306a23abf82b9840738f72c8/helper/schema/resource_data.go#L518-L520) since there's no user provided value. This will cause the request to Keycloak to have `config.importEnabled` set to false, and Keycloak won't supply the default value of true. 

Running a stripped down version of the example on master results in `config.importEnabled` being true, running it on this branch sets it to false.

```hcl
provider "keycloak" {
  client_id     = "terraform"
  client_secret = "884e0f95-0f42-4a63-9b1f-94274655669e"
  url           = "http://localhost:8080"
}

resource "keycloak_realm" "test" {
  realm                = "test"
  enabled              = true
  display_name         = "foo"

  account_theme        = "base"

  access_code_lifespan = "30m"
}

resource "keycloak_ldap_user_federation" "openldap" {
  name                    = "openldap"
  realm_id                = "master"

  enabled                 = true

  username_ldap_attribute = "cn"
  rdn_ldap_attribute      = "cn"
  uuid_ldap_attribute     = "entryDN"
  user_object_classes     = [
    "simpleSecurityObject",
    "organizationalRole"
  ]
  connection_url          = "ldap://openldap"
  users_dn                = "dc=example,dc=org"
  bind_dn                 = "cn=admin,dc=example,dc=org"
  bind_credential         = "admin"
}
```